src/utils.ts

```diff
@@ -148,3 +149,31 @@ export function tokenToSplMint(token: string) {
   }
   return undefined;
 }
+
+export function tokenToPythSymbol(token: string) {
+  if (token == "SOL") {
+    return 'Crypto.SOL/USD';
+  }
+  if (token == "BTC") {
+    return 'Crypto.BTC/USD';
+  }
+  if (token == "ETH") {
+    return 'Crypto.ETH/USD';
+  }
+  return undefined;
+}
+
+export async function getPythPrice(splMint: PublicKey) {
```

Comment:
maybe best to just use mango prices here so we don't have multiple sources? even though I believe fairValue in scalper is using pyth oracle prices

Reply:
Underlying fairvalue in scalper is using mango prices, but the DIP code that pays to the user in this case is using pyth. So at some point in the chain we need to switch from pyth to mango. So this could go either way. Unless there is a way to get mango without a bulky initializing of the mango cache, best to stay with pyth here